src/mysql/create.sql

```diff
@@ -27,13 +27,20 @@ CREATE TABLE User (
 
 CREATE TABLE Notification (
   id INT NOT NULL AUTO_INCREMENT,
-  user_id INT NOT NULL,
   message TEXT,
   url VARCHAR(255),
   date_time DATETIME,
-  PRIMARY KEY (id),
-  FOREIGN KEY (user_id) 
-  REFERENCES User (id)
+  PRIMARY KEY (id)
+);
+
```

Comment:
Consider adding comments that describe the purpose of each of these tables - can be short and sweet.

Reply:
Ok, I will! But I'll make all the SQL changes in another PR, I just had to make small changes to the database for my feature to work.